tiledb/cloud/pool_manager_wrapper.py

```diff
@@ -0,0 +1,10 @@
+class PoolManagerWrapper:
+    def __init__(self, pool_manager):
+        self._pool = pool_manager
+
+    def request(self, method, url, **kwargs):
+        # https://github.com/urllib3/urllib3/issues/2475
+        kwargs["retries"] = self._pool.connection_pool_kw.get("retries")
```

Comment:
This seems like it should be

```
kwargs.setdefault("retries", ...)
```

so that if the caller were to pass their own `retries`, it would not get clobbered.

Reply:
The caller should not be passing `retries` here though. `retries` are passed in the PoolManager constructor, but not propagated here due to https://github.com/urllib3/urllib3/issues/2475

Reply:
That's just a default; a caller can override the behavior for a given request by passing it here (which is in fact exactly what we are doing here, just that the "override" we are passing to the PoolManager is just what the default *should* have been).

Reply:
I understand how it works. What I mean though is that we should not allow the caller to override this but always use the retries passed to the PoolManager constructor for consistency. Once this is fixed upstream, we can remove this functionality completely.

Reply:
My concern here is that this would replace one unwanted behavior (not using the configured retries) with a different unwanted behavior (discarding call-specific retries). When urllib3 itself is fixed, the behavior will be like the `setdefault` behavior, since the default will be used but the caller will be able to override it. Admittedly, this is not a huge concern right now since we don't use it, but I think the behavior we implement here should be as close to The Right Thing as possible: I don't want us to either depend upon the "forbid overrides" behavior (which is unlike the rest of urllib3) or run into a problem where we *do* want to specify a custom retry policy for a specific request.